### PR TITLE
Updated examples

### PR DESCRIPTION
**Pull request by martinrubi**

### CODE REVIEW

**Comment by assislucas** [comment]
@martinrubi Apparently there are conflicts can you fix them please?

**Comment by martinrubi** [comment]
@assislucas Done

File: examples/estimate-shipping-prices.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    return array(
+        'delivery_window' => $delivery_window,
+        'min' => $min,
+        'max' => $max,
+        'average' => $sum / count($eligible_rates),
+    );
+}
+
+// Show estimations for each delivery window
+foreach ($delivery_windows as $delivery_window) {
```
Comment (assislucas) [inline examples/estimate-shipping-prices.php]: I ran this code and for some reason the Min does not print correctly. here's the output I get:

```
PHP Notice:  Undefined offset: 0 in /home/lucas/shippo-wrappers/php-sample-project/estimate.php on line 112
For a delivery window of 1 days:
	Min. costs: 
	Max. costs: 99.58
	Avg. costs: 36.642727272727
PHP Notice:  Undefined offset: 0 in /home/lucas/shippo-wrappers/php-sample-project/estimate.php on line 112
For a delivery window of 3 days:
	Min. costs: 
	Max. costs: 99.58
	Avg. costs: 32.368461538462
For a delivery window of 7 days:
	Min. costs: 5.81
	Max. costs: 99.58
	Avg. costs: 25.94347826087

```

Can you please investigate @martinrubi ?

Comment (martinrubi) [inline examples/estimate-shipping-prices.php]: Fixed.
Not sure if we should use the array_filter function or a regular loop instead.

**Pull request merged**